x-pack/test/saved_object_api_integration/common/fixtures/namespace_agnostic_type_plugin/server/index.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { CoreSetup, Plugin } from 'src/core/server';
+import { PluginSetupContract as FeaturesPluginSetupContract } from '../../../../../../plugins/features/server';
+
+export const plugin = () => new NamespaceAgnosticTypePlugin();
+
+interface SetupDeps {
+  features: FeaturesPluginSetupContract;
+}
+
+class NamespaceAgnosticTypePlugin implements Plugin {
+  setup(core: CoreSetup, plugins: SetupDeps) {
```

Comment:
@elastic/kibana-platform that's another big thing to get rid of LP: to make all tests KP-compatible. we should have done it as a part of the plugin migration effort 😅